html-forms/index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Document</title>
+  </head>
+  <body>
+  <h1>Coding Form!</h1>
+    <form action="www.codingform.com/why.php">
+      <p>
+        <label for="email">Email</label>
+        <input type="email" name="email"/>
+      </p>
+      <p>
+        <label for="coding-is-cool">Why is coding cool?</label>
+        <select name="coding-is-cool">
+          <option value="Select an option">Select an option</option>
+          <option value="Because it's powerful">Because it's powerful</option>
+          <option value="Because it's interesting">Because it's interesting</option>
+          <option value="Because it's fun">Because it's fun</option>
+        </select>
+      </p>
+      <p>Favorite Language (choose one)
+        <input type="radio" value="HTML" name="favorite-lang" id="HTML"/>
+        <label for="HTML">HTML</label>
+        <input type="radio" value="CSS" name="favorite-lang" id="CSS"/>
+        <label for="CSS">CSS</label>
+        <input type="radio" value="JavaScript" name="favorite-lang" id="JavaScript"/>
+        <label for="JavaScript">JavaScript</label>
+      </p>
+      <p>Would you like to sign up for more information?
+        <input type="checkbox" value="subscribe" id="subscribe" name="subscribe"/>
+        <label name="subscribe">Sign me up!</label>
```

Comment:
Same here, this is not configured correctly to pair with its form control element